packages/react-components/theme-designer/src/Context/ThemeDesignerContext.tsx

```diff
@@ -0,0 +1,196 @@
+          createDarkThemeWithUpdatedMapping(state.brand),
+          state.brand,
+        ),
+        darkBrandOverrides: getOverridableTokenBrandColors(createDarkThemeWithUpdatedMapping(state.brand), state.brand),
+      };
+    case 'updateThemeWithCustomerAttributes':
+      const newBrand = createCustomTheme(action.payload);
+      const newTheme = state.isDark ? createDarkThemeWithUpdatedMapping(newBrand) : createLightTheme(newBrand);
+      return {
+        ...state,
+        keyColorHex: action.payload.keyColor,
+        brand: newBrand,
+        theme: newTheme,
+        themeWithOverrides: newTheme,
+        lightThemeOverrides: {},
+        darkThemeOverrides: {},
+        // new overrides -- reset
+        lightBrandOverrides: getOverridableTokenBrandColors(createLightTheme(newBrand), newBrand),
+        darkBrandOverrides: getOverridableTokenBrandColors(createDarkThemeWithUpdatedMapping(newBrand), newBrand),
+      };
+    case 'addOverride':
+      if (state.isDark) {
+        const overrides = {
+          ...state.darkThemeOverrides,
+          // This overrides a THEME token to a new color
+          [action.payload.colorToken]: action.payload.newColor,
+        };
+        return {
+          ...state,
+          darkBrandOverrides: {
+            ...state.darkBrandOverrides,
+            // this is signifying for e.g. 'colorBrandBackground2' = 8 (which is the 8th sqaure in the color wheel)
+            [action.payload.colorToken]: action.payload.brand,
+          } as ColorOverrideBrands,
+          darkThemeOverrides: overrides,
+          themeWithOverrides: {
+            ...createDarkThemeWithUpdatedMapping(state.brand),
+            ...overrides,
+          },
+        };
+      } else {
+        const overrides = {
+          ...state.lightThemeOverrides,
+          // This overrides a THEME token to a new color
+          [action.payload.colorToken]: action.payload.newColor,
+        };
+        return {
+          ...state,
+          lightBrandOverrides: {
+            ...state.lightBrandOverrides,
+            // this is signifying for e.g. 'colorBrandBackground2' = 8 (which is the 8th sqaure in the color wheel)
+            [action.payload.colorToken]: action.payload.brand,
+          } as ColorOverrideBrands,
+          lightThemeOverrides: overrides,
+          themeWithOverrides: {
+            ...createLightTheme(state.brand),
+            ...overrides,
+          },
+        };
+      }
+    case 'themeName':
+      return {
+        ...state,
+        themeName: action.payload,
+      };
+    case 'showExportPanel':
+      return {
+        ...state,
+        showExportPanel: action.payload,
+      };
+
+    default:
+      return {
+        ...state,
+      };
+  }
+};
+
+export const useThemeDesigner = () => {
+  const context = React.useContext(ThemeDesignerContext);
+  if (context === undefined) {
+    throw new Error('useWizard must be used within a WizardProvider');
```

Comment:
What is `useWizard`? I couldn't find any other reference of that in this PR

Reply:
changed this to useThemeDesigner